### ISSUE TITLE
`Symbol.asyncDispose` vs `.disposeAsync()` - why names are inconsistent?

### ISSUE THREAD

**Issue by zloirock** [tc39/proposal-explicit-resource-management#128]
^

**Comment by rbuckton** [comment]
It is often a common practice to have string-named async methods have a trailing `Async` in the name (i.e., [`Atomics.waitAsync`](https://github.com/tc39/proposal-atomics-wait-async), [`Array.fromAsync`](https://github.com/tc39/proposal-array-from-async), etc.). For the symbol name, I chose to match precedent of `[Symbol.iterator]()` and `[Symbol.asyncIterator]()`.

**Comment by zloirock** [comment]
I agree that this convention is a good option in most cases, but here it's methods of one object and it looks weird.

Anyway, it's not something critical for me.
